test/Project.toml

```diff
@@ -1,6 +1,7 @@
 [deps]
 Adapt = "79e6a3ab-5dfb-504d-930d-738a2a938a0e"
 BFloat16s = "ab4f0b2a-ad5b-11e8-123f-65d77653426b"
+CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
```

Comment:
Shouldn't be needed?

Reply:
Feel free to remove, yeah

Reply:
gone now

Reply:
You now removed it from the custatevec project, where we need the module to print the versioninfo. I commented here because you added CUDA as a test dep to CUDA.jl's own tests, which I'm not sure why (I guess it wasn't intended).

Reply:
lmao whoops sorry

Reply:
I think bc I had the wrong `test pkg>` activated :grimacing:

Reply:
Yeah it's a bit tricky with the subprojects. I just have CUDA.jl devved in the subproject's environment. That just adds a line to the (untracked) Manifest in that folder, and makes `Pkg.test()` work as expected.